patch/series_arm64

```diff
@@ -0,0 +1,98 @@
+# This series applies on GIT commit 18c5597832fcf6988111b05a9a1607ae148723c
+0001-i2c-mlxcpld-add-master-driver-for-Mellanox-systems.patch
+0002-i2c-mux-mlxcpld-add-driver-for-Mellanox-systems.patch
+0003-platform-mellanox-Introduce-Mellanox-hardware-platfo.patch
+0004-platform-x86-Introduce-support-for-Mellanox-hotplug-.patch
+0005-leds-add-driver-for-support-Mellanox-regmap-LEDs-for.patch
+0006-Mellanox-switch-drivers-changes.patch
+0007-hwmon-pmbus-Add-support-for-Intel-VID-protocol-VR13.patch
+0008-hwmon-pmbus-Add-support-for-Texas-Instruments-tps536.patch
+0009-platform-mellonox-introduce-mlxreg-io-driver-and-add.patch
+#0010-config-mellanox-configuration.patch
+0011-support-Broadcom-54616-Phy-for-Intel-igb-driver.patch
+0012-platform-mellanox-mlxreg-hotplug-driver-add-check-fo.patch
+driver-arista-net-tg3-dma-mask-4g-sb800.patch
+driver-arista-net-tg3-disallow-broadcom-default-mac.patch
+driver-arista-net-tg3-access-regs-indirectly.patch
+driver-arista-disable-smbus-mux-for-hudson2.patch
+driver-support-sff-8436-eeprom.patch
+driver-support-sff-8436-eeprom-update.patch
+driver-sff-8436-use-nvmem-framework.patch
+driver-sff-8436-use-nvmem_device_read.patch
+driver-support-sff-8436-read-write-fix.patch
+driver-i2c-bus-intel-ismt-add-delay-param.patch
+driver-pca954x-i2c-mux-force-deselect-on-exit-flag.patch
+driver-l3mdev-cgroup.patch
+kernel-add-kexec-reboot-string.patch
+driver-hwmon-max6620.patch
+driver-hwmon-max6620-fix-rpm-calc.patch
+driver-hwmon-max6620-update.patch
+driver-hwmon-max6658-fix-write-convrate.patch
+driver-hwmon-pmbus-dni_dps460.patch
+driver-hwmon-pmbus-dni_dps460-update-pmbus-core.patch
+driver-hwmon-pmbus-dps1900.patch
+driver-support-tun-config-carrier-enable.patch
+driver-support-optoe.patch
+driver-support-optoe-EOF_fix.patch
+driver-support-optoe-chunk-offset-fix.patch
+driver-support-optoe-QSFP_DD.patch
+bridge-add-per-port-broadcast-flood-flag.patch
+driver-net-tg3-add-param-short-preamble-and-reset.patch
+#config-dell-s6000.patch
+#config-dell-z9100.patch
+#config-ingrasys-s9100.patch
+#config-arista-dps1900.patch
+#config-arista-7060-cx32s.patch
+#config-mitac-ly1200.patch
+#config-optoe.patch
+#config-l3mdev-cgroup.patch
+#config-inventec-d7032.patch
+#config-cig-cs6436-serail_new.patch
+0014-mlxsw-qsfp_sysfs-Support-CPLD-version-reading-based-.patch
+0015-platform-x86-mlx-platform-Add-support-for-new-msn201.patch
+0016-platform-mellanox-mlxreg-hotplug-add-extra-run-cycle.patch
+0017-platform-mellanox-add-new-OEM-system-types-to-mlx-pl.patch
+0018-platform-x86-mlx-platform-fix-module-aliases.patch
+0019-platform-x86-mlx-platform-Add-bus-detection-and-diff.patch
+0020-i2c-busses-Add-capabilities-to-i2c-mlxcpld.patch
+linux-4.13-thermal-intel_pch_thermal-Fix-enable-check-on.patch
```

Comment:
now we need to maintain three files at the same time. can you find a way to use one file.

Reply:
kernel .config file 
debian/build/build_amd64_none_amd64/.config
debian/build/build_armhf_none_armmp/.config
debian/build/build_arm64_none_arm64/.config

gets generated after 
fakeroot make -f debian/rules.gen setup_<arch>...

Approach 1:
we can try make debian/rules.gen for all supported arch, so that all .config will be generated.
And then we can maintain a single patch series file, instead maintaining separate files.

Approach 2:
Find a way to skip patch failures in stg import command.